### ISSUE TITLE
WER of Citrinet 256 is not improving.

### ISSUE THREAD

**Issue by Varuzhan97** [NVIDIA/NeMo#7789]
Hi. I have 34 hours of ASR dataset (split in an 80-20 ratio for training and validation) in the Armenian language and want to train the ASR model using Citrinet 256 (transfer-learning from stt_en_citrinet_256). After some epochs, WER is going up. How to improve this or must I use Citrinet 512? Below are hyperparameters of training:
`tokenizer type is unigram,`
`vocab size is 1024,`
`freeze_encoder: False,`
`batch_size: 4,`
`use_start_end_token: True,`
`trim_silence: False,`
`learning rate: 0.001,`
`betas: [0.95, 0.5],`
`weight_decay: 0.001,`
`warmup_steps: None,`
`warmup_ratio: 0.1,`
`minimum learning rate: 1e-5,`
`spec_augment.freq_masks: 2,`
`spec_augment.freq_width: 27,`
`spec_augment.time_masks: 10,`
`spec_augment.time_width: 0.05,`
`epochs: 200,`

**Comment by titu1994** [comment]
Model is probably overfitting to small amount of training data. You can train for shorter duration, use lower lr or change model to something larger that can learn better. 

You can try Conformer small, it has similar parameters count as citrinet 256.

**Comment by titu1994** [comment]
Not really. You'll need to play with spec augment, lr, longer or shorter training runs etc. Small models on small datasets overfit super fast

**Comment by Varuzhan97** [comment]
ok, thanks a lot.
